technical_standards/testing/index.md

```diff
@@ -0,0 +1,38 @@
+# Testing
+
+While automated testing is often praised, it is important to remember that they are just a tool with their pros and cons.
+
+That is why it is important to get them right to not end up with a burden slowing your developing workflow.
+
+## Testing everything you should not
+
+Automating a test is not mandatory, and there are cases there you should not automate them.
```

Comment:
Automating a test is not mandatory, and there are cases there you should not automate.